src/chrome/touch_handler.js

```diff
@@ -0,0 +1,43 @@
+var touchstartX = 0;
+var touchstartY = 0;
+var touchendX = 0;
+var touchendY = 0;
+
+var gesuredZone = document;
+
+gesuredZone.addEventListener('touchstart', function(event) {
+    console.log(event);
+    console.log(touchstartX = event.changedTouches[0].screenX);
+    console.log(touchstartY = event.changedTouches[0].screenY);
+}, false);
+
+gesuredZone.addEventListener('touchend', function(event) {
+    console.log(event);
+    console.log(touchendX = event.changedTouches[0].screenX);
+    console.log(touchendY = event.changedTouches[0].screenY);
+    handleGesure();
+}, false); 
+
+function handleGesure() {
+    var swiped = 'swiped: ';
+    if ((touchendX - touchstartX) <= -130) {
```

Comment:
Just wondering, where did the number `130` come from? Is it standardized for swipes or is it just a random number that feels "right"?

Reply:
>  just a random number that feels "right"
Yes... That's just a threshold that I found when randomly testing it, I'm open to suggestion though, you can test it yourself